terraform/outputs.tf

```diff
@@ -4,6 +4,7 @@ output "default_apigatewayv2_stage_invoke_url" {
 
 output "api_gw_api_key" {
   value = aws_api_gateway_api_key.kuberoke.value
+  sensitive = true
```

Comment:
I guess we can keep this as is or add the flag, doesn't make a big difference from my POV.

Reply:
Yes, the API key needs to go into the HTML template indeed.

Reply:
I removed it and added instructions for how to find the key in the console.